.github/workflows/main.yaml

```diff
@@ -236,7 +233,7 @@ jobs:
   ########################
   integration-test-postgres:
     name: run itests postgres
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
We do we need to change this?

And, if it needs to be updated, shouldn't we change it to a fixed version so that it doesn't potentially stop working at some point in the future?

Reply:
We've been running with this in our CI for a very long time now (in lnd) and it never lead to any issues. Downgrading was just a previous attempt to fix this problem (since apparently older images were less prone to kill on out of memory).